src/stores/systemPreferences.ts

```diff
@@ -70,15 +70,20 @@ const preferences = [
   // {preference: 'com.apple.Xsan'},
 ]
 
-export function buildSystemPreferenceItem(item: {
-  preference: string
-  icon?: string
-  name?: string
-}) {
+export function buildSystemPreferenceItem(
+  item: {
+    preference: string
```

Comment:
rename this to `path`?

Reply:
what I mean is: what is this supposed to mean? a URL? a panel identifier? it's not clear from such a generic name

Reply:
agreed!